sdk/keyvault/keyvault-secrets/test/utils/recorderUtils.ts

```diff
@@ -20,3 +20,7 @@ export function uniqueString(): string {
         .toString()
         .slice(2);
 }
+
+export const testPollerProperties = {
+  intervalInMs: isRecording ? 2000 : 0
```

Comment:
Do we need the full 2000 ms for live tests as well? We should avoid intentional delays as much as possible to avoid excessively long test running.

Reply:
the service takes considerably above 2 seconds to respond. Sometimes it's 20 seconds or more.

Reply:
wow that seems very high, seems like we have some room to improve.